build.gradle

```diff
@@ -564,6 +564,7 @@ project(':core') {
     testCompile libs.easymock
     testCompile(libs.apacheda) {
       exclude group: 'xml-apis', module: 'xml-apis'
+      exclude module: 'mina-core'
```

Comment:
Can we please add a comment explaining the exclusion?

Reply:
@ijuma Thanks for the review, added comment.